libcontainer/intelrdt/intelrdt.go

```diff
@@ -478,6 +478,16 @@ func (m *Manager) Apply(pid int) (err error) {
 		return newLastCmdError(err)
 	}
 
+	// Create MON group
+	if monPath := m.GetMonPath(); monPath != "" {
+		if err := os.Mkdir(monPath, 0o755); err != nil && !os.IsExist(err) {
```

Comment:
Who has control of this path? In runc this is trusted, okay, but is it exposed in k8s or containerd or some other to the user?

Not sure with the https://github.com/intel/k8s-rdt-controller what is exposed to an end user

Reply:
That, in turn, comes from parsing the output of statfs syscall (`unix.Statfs()`). Note that in the case of runc update the closPath is taken from the `config.json` of the container.

In any case I cannot see any way that an unprivileged user can control the path